--- vorton/Makefile.linux.ORIGINAL   2016-03-30 22:45:18.075043520 +0000
+++ vorton/Makefile.linux   2016-03-30 22:49:14.582265492 +0000
@@ -5,7 +5,7 @@
 STRIP=      strip
 
 
-CFLAGS =   -Wall -Wno-switch -march=pentium -mcpu=pentium -DDEBUG \
+CFLAGS =   -Wall -Wno-switch -DDEBUG \
       -DDATA_PREFIX=\"data/\" -DFILE_SEPARATOR=\"/\" -DARGS \
       -Isrc -Isrc/SDL_flic -Isrc/SFont `sdl-config --cflags` \
       -ffast-math -fno-common -ffreestanding -fno-builtin \
@@ -16,7 +16,7 @@
 #CFLAGS+=-DUSE_FIXED_POINT
 
 
-LDFLAGS =   `sdl-config --libs` -lSDL_mixer -lSDL_image \
+LDFLAGS =   `sdl-config --libs` -lSDL_mixer -lSDL_image -lm
 
 
 .CFILES   =   src/vorton.c \
